reactjs/src/scenes/Login/index.validation.ts

```diff
@@ -0,0 +1,3 @@
+// const rules = [];
+
+// export default rules;
```

Comment:
rules are defined in `scenes/Login/index.tsx`, so no need to keep this file.

Reply:
Resolved